setup.py

```diff
@@ -13,7 +13,7 @@ def read_version(package):
 VERSION = read_version("iolite")
 
 setup(
-    description="API client for interacting with the https://remote.iolite.com/ API",
+    description="",
```

Comment:
I understand you moved the description to the `pyproject.toml`. Should we remove the description here, then, instead of leaving it blank?

Reply:
Whoops - that was a mistake. Do we even need this file anymore?